File: studyapp2/app/src/main/java/com/example/studyapp/MainActivity.java
package com.example.studyapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import com.google.firebase.firestore.FirebaseFirestore;

public class MainActivity extends AppCompatActivity {
    public static final String EXTRA_NUMBER = "package com.example.study_application";
    private static final String EXTRA_PASSWORD = "pass";
    private static final String EXTRA_USERNAME = "user";

    FirebaseFirestore db = FirebaseFirestore.getInstance();

    EditText Password;
    EditText Username;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Username = findViewById(R.id.Username);
        Password = findViewById(R.id.Password);


        Button SignIn = findViewById(R.id.SignIn);
        SignIn.setOnClickListener(v -> signIn());
    }

    private void signIn(){
        System.out.println(Password.toString());
        Intent toHomeScreen = new Intent(this, HomeScreen.class);
        toHomeScreen.putExtra(EXTRA_PASSWORD,Password.getText().toString());
        toHomeScreen.putExtra(EXTRA_USERNAME,Username.getText().toString());
        startActivity(toHomeScreen);
    }
}

File: MyApplicationkotlin/app/src/main/java/com/example/myapplicationkotlin/MainActivity.kt
package com.example.myapplicationkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ProgressBar
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    private var progr = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        updateProgressBar()
        val button_incr = findViewById<Button>(R.id.button_incr)
        val button_decr = findViewById<Button>(R.id.button_decr)


        button_incr.setOnClickListener {
            if (progr <= 90) {
                progr += 10
                updateProgressBar()
            }
        }

        button_decr.setOnClickListener {
            if (progr >= 10) {
                progr -= 10
                updateProgressBar()
            }
        }
    }

    private fun updateProgressBar() {
        val progress_bar = findViewById<ProgressBar>(R.id.progress_bar)
        val text_view_progress = findViewById<TextView>(R.id.text_view_progress)

        progress_bar.progress = progr
        text_view_progress.text = "$progr%"
    }
}